src/game/common/globaldata.cpp

```diff
@@ -34,350 +34,672 @@ GlobalData *g_theWriteableGlobalData = nullptr;
+    { "StructureStiffness", &INI::Parse_Real, nullptr, offsetof(GlobalData, m_structureStiffness) }, //
+    { "Gravity", &INI::Parse_Acceleration_Real, nullptr, offsetof(GlobalData, m_gravity) }, //
+    { "StealthFriendlyOpacity", &INI::Parse_Percent_To_Real, nullptr, offsetof(GlobalData, m_stealthFriendlyOpacity) }, //
+    { "DefaultOcclusionDelay",
+        &INI::Parse_Duration_Unsigned_Int,
+        nullptr,
+        offsetof(GlobalData, m_defaultOcclusionDelay) }, //
+    { "PartitionCellSize", &INI::Parse_Real, nullptr, offsetof(GlobalData, m_partitionCellSize) }, //
+    { "AmmoPipScaleFactor", &INI::Parse_Real, nullptr, offsetof(GlobalData, m_ammoPipScaleFactor) }, //
+    { "ContainerPipScaleFactor", &INI::Parse_Real, nullptr, offsetof(GlobalData, m_containerPipScaleFactor) }, //
+    { "AmmoPipWorldOffset", &INI::Parse_Coord3D, nullptr, offsetof(GlobalData, m_ammoPipWorldOffset) }, //
+    { "ContainerPipWorldOffset", &INI::Parse_Coord3D, nullptr, offsetof(GlobalData, m_containerPipWorldOffset) }, //
+    { "AmmoPipScreenOffset", &INI::Parse_Coord2D, nullptr, offsetof(GlobalData, m_ammoPipScreenOffset) }, //
+    { "ContainerPipScreenOffset", &INI::Parse_Coord2D, nullptr, offsetof(GlobalData, m_containerPipScreenOffset) }, //
+    { "HistoricDamageLimit", &INI::Parse_Duration_Unsigned_Int, nullptr, offsetof(GlobalData, m_historicDamageLimit) }, //
+    { "MaxTerrainTracks", &INI::Parse_Int, nullptr, offsetof(GlobalData, m_maxTerrainTracks) }, //
+    { "TimeOfDay", &INI::Parse_Index_List, g_timeOfDayNames, offsetof(GlobalData, m_timeOfDay) }, //
+    { "Weather", &INI::Parse_Index_List, g_weatherNames, offsetof(GlobalData, m_weather) }, //
+    { "MakeTrackMarks", &INI::Parse_Bool, nullptr, offsetof(GlobalData, m_makeTrackMarks) }, //
+    { "HideGarrisonFlags", &INI::Parse_Bool, nullptr, offsetof(GlobalData, m_hideGarrisonFlags) }, //
+    { "ForceModelsToFollowTimeOfDay", &INI::Parse_Bool, nullptr, offsetof(GlobalData, m_forceModelsFollowTimeOfDay) }, //
+    { "ForceModelsToFollowWeather", &INI::Parse_Bool, nullptr, offsetof(GlobalData, m_forceModelsFollowWeather) }, //
+    { "LevelGainAnimationName", &INI::Parse_AsciiString, nullptr, offsetof(GlobalData, m_levelGainAnimName) }, //
+    { "LevelGainAnimationTime", &INI::Parse_Real, nullptr, offsetof(GlobalData, m_levelGainAnimTime) }, //
+    { "LevelGainAnimationZRise", &INI::Parse_Real, nullptr, offsetof(GlobalData, m_levelGainAnimZRise) }, //
+    { "GetHealedAnimationName", &INI::Parse_AsciiString, nullptr, offsetof(GlobalData, m_getHealedAnimName) }, //
+    { "GetHealedAnimationTime", &INI::Parse_Real, nullptr, offsetof(GlobalData, m_getHealedAnimTime) }, //
+    { "GetHealedAnimationZRise", &INI::Parse_Real, nullptr, offsetof(GlobalData, m_getHealedAnimZRise) }, //
+    // Light 1
+    { "TerrainLightingMorningAmbient",
+        &INI::Parse_RGB_Color,
+        nullptr,
+        offsetof(GlobalData, m_terrainPlaneLighting[TIME_OF_DAY_MORNING][0].ambient) },
```

Comment:
Difficult to review these when they are not collapsed into one line. Maybe use `// clang-format off` comment?

Reply:
i actually think this is more readable than a oneliner - it's also formatted like this in other places